### ISSUE TITLE
aca_view in ska3-next still needs env QT_MAC_WANTS_LAYER=1

### ISSUE THREAD

**Issue by taldcroft** [sot/skare3#789]
**Description**
Running `aca_view` in ska3-next rc6 still does not produce a window on Mac without the `env QT_MAC_WANTS_LAYER=1` hack.

**Comment by javierggt** [comment]
I thought this was not an issue for you when I asked.

**Comment by jeanconn** [comment]
I thought we decided it was still an issue, just not one that precluded promotion?  And is it a skare3 issue or an aca_view issue? And should it be fixed with documentation, a sh script,  something set in the aca_view startup..., .. an option someplace in a qt config file?

**Comment by jeanconn** [comment]
The internet suggests this is fixed in pyqt >= 5.15.2 ?

**Comment by taldcroft** [comment]
Agreed this is not a blocker for promotion of ska3-next, but it does need to be called out in the FSDS ticket as a known issue. I put this as a skare3 issue for that reason, and maybe it is fixed by pyqt and not via `aca_view` as @jeanconn suggests?

**Comment by jzuhone** [comment]
As previously noted, this is not an issue for `xija_gui_fit`, maybe there is something to be determined from the difference in the Qt setup of the tools.

**Comment by jeanconn** [comment]
This is no longer an issue for me with aca_view and QT_MAC_WANTS_LAYER not set.
